runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/Scheduler.java

```diff
@@ -362,6 +379,7 @@ public synchronized void start() {
           readConfiguration();
         }
         scanLastStarted = Instant.now();
+        UnreadableDirectories unreadableDirectories = new UnreadableDirectories();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "unreadableDirectories" which hides the field declared at line 220. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)